modules/auxiliary/scanner/ssl/openssl_heartbleed.rb

```diff
@@ -109,7 +109,10 @@ def initialize
         STARTTLS may also be vulnerable.
 
         The module supports several actions, allowing for scanning, dumping of
-        memory contents, and private key recovery.
+        memory contents, and private key recovery. The `repeat` command can be
```

Comment:
I was considering an advanced option for all modules for number of times to loop, but this felt more flexible and possibly less obscure. I think the MS15-034 scanner could also benefit.

Reply:
Additionally, I had found it annoying to use the Heartbleed module because you had to check the loot file(s) manually for anything interesting. I would have liked a way to `tee` in a sense, but that was well outside design parameters.